src/locales/en/messages.json

```diff
@@ -1453,9 +1453,15 @@
   "verifyBrowserTitle": {
     "message": "Verify browser connection"
   },
-  "verifyBrowserDescription": {
+  "verifyBrowserDesc": {
```

Comment:
Why the change to an abbreviation? Does it need to change?

Reply:
We use the abbreviation for all the other dialogs, so seemed best to be consistent.